### PR TITLE
Improve batch changes site admin guide

### PR DESCRIPTION
**Pull request by malomarrec**

### CODE REVIEW

**Comment by malomarrec** [comment]
Reviewers: feel free to merge :)

File: doc/batch_changes/how-tos/index.md [hunk 1]
```diff
@@ -8,11 +8,14 @@ The following is a list of how-tos that show how to use [Sourcegraph Batch Chang
...
-- [Site admin configuration for Batch Changes](site_admin_configuration.md)
 - [Configuring credentials for Batch Changes](configuring_credentials.md)
 - [Handling errored changesets](handling_errored_changesets.md)
 - [Opting out of batch changes](opting_out_of_batch_changes.md)
 - [Bulk operations on changesets](bulk_operations_on_changesets.md)
 - Batch changes in monorepos
   - [Creating changesets per project in monorepos](creating_changesets_per_project_in_monorepos.md)
   - <span class="badge badge-experimental">Experimental</span> [Creating multiple changesets in large repositories](creating_multiple_changesets_in_large_repositories.md)
+
+### Batch Changes site admin
```
Comment (LawnGnome) [inline doc/batch_changes/how-tos/index.md:19]:
```suggestion
### Administering Batch Changes
```

File: doc/batch_changes/how-tos/site_admin_configuration.md [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
-Site admins can also:
 
-- [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.
-- [Control the rate at which Batch Changes will create changesets](../../../admin/config/batch_changes.md#rollout-windows).
+#### Setup Batch Changes 
+1. Using Batch Changes requires a [code host connection](../../../admin/external_service/index.md) to a supported code host (currently GitHub, Bitbucket Server, and GitLab).
+1. [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.
+1. [Configure credentials](configuring_credentials.md).
+1. Setup webhooks to make sure changesets sync fast. See [Batch Changes effect on codehost rate limits](../references/requirements.md#batch-changes-effect-on-code-host-rate-limits).
```
Comment (LawnGnome) [inline doc/batch_changes/how-tos/site_admin_configuration.md:9]: We might want to make it clearer that there are step by step instructions available:

```suggestion
1. Setup webhooks to make sure changesets sync fast. See [Batch Changes effect on codehost rate limits](../references/requirements.md#batch-changes-effect-on-code-host-rate-limits), and select your code host to get step-by-step instructions.
```

Or, alternatively, we could just link directly to the code host pages and save the user a click.

File: doc/batch_changes/references/faq.md [hunk 3]
```diff
@@ -82,3 +82,6 @@ Since Sourcegraph 3.31, it is possible to publish many types of changeset when p
 - The changeset was imported (and was therefore already published by someone or something else).
 
 The changeset may also be in a state that we cannot currently publish from: for example, because a previous push to the code host failed (in which case you should re-apply the batch change), or if you are actively detaching the changeset from your batch change.
+
+### Changesets are taking a lot of time to sync
```
Comment (LawnGnome) [inline doc/batch_changes/references/faq.md:86]: For consistency:

```suggestion
### Why do my changesets take a long time to sync?
```

**Review by LawnGnome — APPROVED** [review]

File: doc/batch_changes/how-tos/site_admin_configuration.md [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
 
-Using Batch Changes requires a [code host connection](../../../admin/external_service/index.md) to a supported code host (currently GitHub, Bitbucket Server, and GitLab).
 
-Site admins can also:
 
-- [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.
-- [Control the rate at which Batch Changes will create changesets](../../../admin/config/batch_changes.md#rollout-windows).
+#### Setup Batch Changes 
+1. Using Batch Changes requires a [code host connection](../../../admin/external_service/index.md) to a supported code host (currently GitHub, Bitbucket Server, and GitLab).
+1. [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.
```
Comment (mrnugget) [inline doc/batch_changes/how-tos/site_admin_configuration.md:7]:
```suggestion
1. (Optional) [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.
```

File: doc/batch_changes/how-tos/site_admin_configuration.md [hunk 5]
```diff
@@ -1,11 +1,15 @@
...
+1. [Configure credentials](configuring_credentials.md).
+1. Setup webhooks to make sure changesets sync fast. See [Batch Changes effect on codehost rate limits](../references/requirements.md#batch-changes-effect-on-code-host-rate-limits).
+1. (Optional) [Control the rate at which Batch Changes will create changesets](../../../admin/config/batch_changes.md#rollout-windows).
```
Comment (mrnugget) [inline doc/batch_changes/how-tos/site_admin_configuration.md:10]:
```suggestion
1. (Optional) [Control the rate at which Batch Changes will publish changesets on code hosts](../../../admin/config/batch_changes.md#rollout-windows).
```

**Review by mrnugget — APPROVED** [review]

File: doc/batch_changes/how-tos/site_admin_configuration.md (hunk 2 above)
Comment (malomarrec) [reply doc/batch_changes/how-tos/site_admin_configuration.md:9]: Went with the alternative

**Pull request merged**